cpp/src/slice2cpp/Gen.cpp

```diff
@@ -987,95 +978,6 @@ Slice::Gen::generate(const UnitPtr& p)
         Cpp11CompatibilityVisitor compatibilityVisitor(H, C, _dllExport);
         p->visit(&compatibilityVisitor, false);
     }
-
```

Comment:
You should also remove all these C++98 visitors: ProxyDeclVisitor, DeclVisitor, etc. I am pretty sure they are independent of the Cpp11 visitors.